Changelog

```diff
@@ -95,7 +95,7 @@ Thu Jan 06 2022 - sledgehammer999 <sledgehammer999@qbittorrent.org> - v4.4.0
     - FEATURE: Add tooltip to listening port spinbox (Chocobo1)
     - FEATURE: Add "Skip hash check" option for watched folders (glassez)
     - FEATURE: Add "Show torrent options" double-click action (glassez)
-    - FEATURE: Allow setting temp folder per torrent/catergory (glassez)
+    - FEATURE: Allow setting temp folder per torrent/category (glassez)
```

Comment:
IIRC sledgehammer999 would like to keep it as is for the sake of users to link to the commit message.
And personally I'm ok either way.